src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -192,8 +192,11 @@ class JsonRpcController(
     case req @ JsonRpcRequest(_, "personal_ecRecover", _, _) =>
       handle[EcRecoverRequest, EcRecoverResponse](personalService.ecRecover, req)
 
-    case req @ JsonRpcRequest(_, "personal_deleteWallet", _, _) =>
+    case req @ JsonRpcRequest(_, "daedalus_deleteWallet", _, _) =>
```

Comment:
Minor detail: Should we maybe move this and the handling of `daedalus_changePassphrase` to a function like `handleDaedalusRequest`?

Reply:
Good point, and also I think I should add it [here](https://github.com/input-output-hk/mantis/blob/68a5e642bf3056bbd179c2cd7fc03d7fc5b92164/src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala#L30) and make it configurable